WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -14,15 +14,12 @@ class MediaCoordinator: NSObject {
 
     private let queue = DispatchQueue(label: "org.wordpress.mediauploadcoordinator")
 
-    private lazy var mediaProgressCoordinator: MediaProgressCoordinator = {
+    private(set) lazy var mediaProgressCoordinator: MediaProgressCoordinator = {
         let coordinator = MediaProgressCoordinator()
         coordinator.delegate = self
         return coordinator
     }()
 
-    // Init marked private to ensure use of shared singleton.
-    private override init() {}
```

Comment:
We discussed this separately, but just leaving a note here – I think we'll probably want to switch back to a singleton after we can leave the editor during uploads.